packages/peregrine/lib/talons/CartPage/PriceAdjustments/useCouponCode.js

```diff
@@ -30,10 +30,11 @@ export const useCouponCode = props => {
         async ({ couponCode }) => {
             if (!couponCode) return;
             try {
+                // Trim whitespace before submitting the coupon code
                 await applyCoupon({
                     variables: {
                         cartId,
-                        couponCode
+                        couponCode: couponCode.trim()
```

Comment:
I would prefer to see this behavior prevented in the UI, not applied before the network request. Add `mask={value => value && value.trim()}` to `couponCode.js`, and this shouldn't be needed. If you can think of an edge case where this would still be needed, I'm fine leaving it around.

Reply:
Oh...that's much cleaner :D